corehq/apps/ota/tests/test_search_claim_endpoints.py

```diff
@@ -1,14 +1,16 @@
 import re
-from uuid import uuid4
+from uuid import uuid1, uuid4
 
 from django.test import Client, TestCase
 from django.urls import reverse
+from custom.covid.tests.test_prime_restore import make_synclog
```

Comment:
This utils method doesn't really do very much and it's in a custom module so not great to import into core code.

I think it would be fine to make a local function that applies any defaults for these tests.

If the existing function contained logic or something more complex then I'd suggest moving it to a more central location (like somewhere alongside the Synclog model itself).

Reply:
That makes sense. I added a custom make_synclog() function to CaseClaimEndpointTests in a new commit, can you give it a look?